src/client/components/Modals.tsx

```diff
@@ -3,20 +3,28 @@ import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
 import Modal from '@material-ui/core/Modal';
 import Backdrop from '@material-ui/core/Backdrop';
 import Fade from '@material-ui/core/Fade';
-import { Button, Grid, Paper, TextField } from '@material-ui/core';
+import { Button, Grid, IconButton, Paper, SvgIcon, SvgIconProps, TextField } from '@material-ui/core';
```

Comment:
You won't need to import svg stuff, just import the close icon from @material-ui/icons